### PR TITLE
add axel

### PR DESCRIPTION
**Pull request by lizhuoli1126**
axel: add

### CODE REVIEW

File: pages/common/axel.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+- Download and specify filename
+
+`axel {{url}} -o {{filename}}`
+
+- Download with multiple connections
+
+`axel -n {{connections_num}} {{url}}`
+
+- Search for mirrors (can specify numbers)
```
Comment (felixonmars) [inline pages/common/axel.md]: I think "(can specify numbers)" can be omitted, since there's `{{mirrors_num}}` in the example already.

**Comment by G570972680893** [comment]
😎

File: pages/common/axel.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`axel -S {{mirrors_num}} {{url}}`
+
+- Limit download speed (bytes per second)
+
+`axel -s {{speed}} {{url}}`
```
Comment (G570972680893) [inline pages/common/axel.md]: 😎

File: pages/common/axel.md (hunk 1 above)
Comment (lizhuoli1126) [inline pages/common/axel.md]: I means the default mirrors_num is 1 if you just use 'axel -n {{ur}}l'..But I think it may cause misunderstanding

**Comment by rubenvereecken** [comment]
I'm merging this one in but the last example doesn't seem very necessary. If we ever need a different example, that one's the first to go :P.

Thanks!

**Pull request merged**